specs/Main.md

```diff
@@ -40,7 +40,7 @@ A copy of the license is included in the repository, under the file entitled LIC
 | 14  | r14           | General        |                       Fast Indirect Call Address                        |
 | 15  | r15/link      | General        |                        Fast Call Return Address                         |
 | 16  | ip            | Readonly[^1]   |                           Instruction Pointer                           |
-| 17  | flags         | Flags          |                            Processor Status                             |                                                                    |
+| 17  | flags         | Flags          |                            Processor Status                             |
 | 63  | reserved      | Reserved       | Undefined register. Will not be given meaning in a future version |
 | 128 | cr0           | Supervisor     |                            Processor Control                            |
 | 129 | cr1/page      | Supervisor     |            Physical Address of the Virtual Memory Page Table            |
```

Comment:
The line after this line is still broken

Reply:
And by the line after this line, I mean the latest line after the latest line I can select to review.

Reply:
I'm not sure I understand what you mean. There also doesn't seem to be an issue in the rendered text, just the markdown

Reply:
Line 47; there's an extra space after the text "130." It won't show up in the render, but it's obvious in the markdown.